static/app/views/organizationGroupDetails/eventToolbar.tsx

```diff
@@ -110,40 +107,33 @@ class GroupEventToolbar extends Component<Props> {
             location={location}
           />
         </div>
-        <NavigationContainer>
-          {hasReplay && isReplayEnabled ? (
-            <Button href="#breadcrumbs" size="sm" icon={<IconPlay size="xs" />}>
-              {t('Replay')}
-            </Button>
-          ) : null}
```

Comment:
After removing this button, we need to make sure that the trace navigator is rendering the link to the replay in all the same cases.

In the case where: there are no transactions but there is a replay... we should still render the replay link in the trace navigator.